### PR TITLE
Use FHS compliant path (part of fate#325482)

### PR DESCRIPTION
**Pull request by dgdavid**
Related to https://trello.com/c/piNC0YZM/

### CODE REVIEW

**Comment by dgdavid** [comment]
BTW, someone knows why is the syntax verification commented?

https://github.com/yast/skelcd-control-SLES4SAP/pull/12/files#diff-cef63e48012c7373fe2e68f8b0bb631bR80

File: package/skelcd-control-SLES4SAP.changes [hunk 1]
```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Tue Dec 11 17:01:30 UTC 2018 - dgonzalez@suse.com
+
+- Use FHS compliant path (fate#325482)
+- 15.0.6
```
Comment (ancorgs) [inline package/skelcd-control-SLES4SAP.changes]: Same case than https://github.com/yast/skelcd-control-SLE_HPC/pull/19 We should either use version 15.1.0 or submit this also to the SLE-15-GA branch.

File: package/skelcd-control-SLES4SAP.spec [hunk 2]
```diff
@@ -27,26 +27,31 @@
...
+
+Name:           skelcd-control-%{skelcd_name}
 # xsltproc for converting SLES control file to SLES-for-VMware
 BuildRequires:  libxslt-tools
 # xmllint (for validation)
 BuildRequires:  libxml2-tools
-# RNG validation schema
-BuildRequires: yast2-installation-control
+# Added skelcd macros
+BuildRequires: yast2-installation-control >= 4.1.5
```
Comment (dgdavid) [inline package/skelcd-control-SLES4SAP.spec:38]: See yast/yast-installation-control#75

**Comment by lslezak** [comment]
> BTW, someone knows why is the syntax verification commented?

https://github.com/yast/skelcd-control-SLES4SAP/commit/b05346b7984ac4cf28092e7d212b2b233c7e9347#diff-cef63e48012c7373fe2e68f8b0bb631bR73

No idea, I guess it was some temporary hack to make it build.

Please enable it back.

File: package/skelcd-control-SLES4SAP.spec [hunk 3]
```diff
@@ -65,10 +70,8 @@ SLES4SAP control file needed for installation
 
 %build
 # transform ("patch") the original SLES installation file
-xsltproc %{SOURCE0} /installation.xml > installation.xml
-#xsltproc %{SOURCE0} /installation.xml > installation1.xml
-#xmllint --format installation1.xml > installation.xml
-diff -u /installation.xml installation.xml || :
+xsltproc %{SOURCE0} %{skelcd_control_datadir}/SLES.xml > installation.xml
+diff -u %{skelcd_control_datadir}/SLES.xml installation.xml || :
```
Comment (dgdavid) [inline package/skelcd-control-SLES4SAP.spec:74]: @lslezak please, take a look to this two lines. Looks good for you?

I could build the package successfully.

File: package/skelcd-control-SLES4SAP.spec [hunk 4]
```diff
@@ -27,26 +27,31 @@
...
-# RNG validation schema
-BuildRequires: yast2-installation-control
+# Added skelcd macros
+BuildRequires: yast2-installation-control >= 4.1.5
 
 # original SLES control file
 BuildRequires: skelcd-control-SLES >= 15.0.0
 BuildRequires: diffutils
 
+# USe FHS compliant path
```
Comment (lslezak) [inline package/skelcd-control-SLES4SAP.spec]: NP: upcase `S`

**Review by lslezak — APPROVED** [review]
LGTM, you just might restart Travis after the SLES skelcd is merged (and built in OBS)

File: package/skelcd-control-SLES4SAP.spec (hunk 4 above)
Comment (dgdavid) [reply package/skelcd-control-SLES4SAP.spec]: :man_facepalming:

**Pull request merged**